.github/workflows/check.yml

```diff
@@ -64,9 +64,7 @@ jobs:
 
     - name: Install packages
       run: |
-        sudo apt install -y nasm gcc g++ gcc-multilib g++-multilib make binutils grub-pc-bin qemu-system-x86 clang-tidy
-        sudo apt install -y build-essential bison flex libgmp3-dev libmpc-dev
-        sudo apt install -y xorriso mtools inkscape libmpfr-dev texinfo libfuse-dev
+        sudo apt install -y nasm gcc gcc-multilib clang-tidy
```

Comment:
Heuuu you just need `clang-tidy` :sweat_smile:  (the rest is already installed above)

Reply:
I need also gcc-multilib in order for clang-tidy not to report some missing headers.